rust/osm2lanes/src/transform/tags_to_lanes/mod.rs

```diff
@@ -179,6 +180,12 @@ impl LaneBuilder {
             None => panic!(),
         }
     }
+
+    /// Create a mirrored version of the lane
+    #[must_use]
+    fn mirror(&self) -> &Self {
+        self
```

Comment:
Shouldn't this call something?

Reply:
I wanted to get the interface in. For now it actually doesn't need to do anything, because none of the lanes are handed (no chirality)

Reply:
```suggestion
        # TODO: this doesn't need to do anything for now
        # check back after v1.0.0 to see if this is still the case
        self
```